hadoop-hdds/docs/content/feature/OM-HA.md

```diff
@@ -125,7 +125,23 @@ ozone om [global options (optional)] --bootstrap --force
 
 Note that using the _force_ option during bootstrap could crash the OM process if it does not have updated configurations.
 
+## Automatic Snapshot Installation for Stale Ozone Managers
+
+Sometimes an OM follower node may be offline or fall so far behind the leader OM's log that it cannot catch up by replaying individual log entries.  The OM HA implementation includes an automatic snapshot installation and recovery process for such cases.
```

Comment:
Since git diff compares files line by line, I suggest using short lines for better tracking.  I also revise some sentenses.

```md
Sometimes an OM follower node may be offline or fall far behind the OM leader's raft log.
Then, it cannot easily catch up by replaying individual log entries.
The OM HA implementation includes an automatic snapshot installation 
and recovery process for such cases.
```

Reply:
Try `git diff --word-diff`, it works well for text and can clarify changes in long lines.

Unless the change is trivial (e.g. fixing a typo), short lines usually either get unbalanced, or rewrapping hides the actual change.  Then `git diff` with or without `--word-diff` becomes less useful.

Example word diff from recent doc change (as a screenshot, because GitHub does not support highlighting word diff properly, but Git does):

![image](https://github.com/user-attachments/assets/b464cc43-cab2-446c-9685-11b77adac1a6)